tests/lib/proc_utils.py

```diff
@@ -10,7 +10,7 @@
 import subprocess
 
 
-def cmd_exec(args, env=None, log=None, verbose=True):
+def cmd_exec(args, env=None, log=None, verbose=True, shell=False):
```

Comment:
where shell=True is used? I can't find it

Reply:
@asomsiko, tests/conditional_compilation/test_utils.py: 79